crates/mun_runtime/src/allocator.rs

```diff
@@ -26,7 +26,7 @@ impl Allocator {
     }
 
     /// Allocates a block of memory
-    fn alloc(&self, size: u64, alignment: u64) -> *mut u8 {
+    fn alloc(&self, size: usize, alignment: usize) -> *mut u8 {
         unsafe {
             std::alloc::alloc(Layout::from_size_align(size as usize, alignment as usize).unwrap())
```

Comment:
`as usize` is no longer necessary. Also, if we always `unwrap`, we might as well use the `_unchecked` version:
```suggestion
            std::alloc::alloc(unsafe { Layout::from_size_align_unchecked(size, alignment) })
```

Reply:
Or we could receive a `Layout` in the `alloc` function.